### ISSUE TITLE
Script for peers to join channel fails in some cases

### ISSUE THREAD

**Issue by dwakeman** [IBM-Blockchain/ibm-blockchain.github.io#52]
I ran `./create_all.sh` today against a clean cluster but the `create/join_channel.sh` script failed.  Inspection of the logs of the `joinchannel` container showed that it could not connect to the peer.  The 15 second delay at the end of `create_blockchain.sh` is not always enough time for the peers to get settled.  I deleted my cluster and created a new one, and also changed that delay to 60 seconds and that time everything worked.

**Comment by mrshah-at-work** [comment]
@dwakeman we can make the delay configurable using environment variables.

**Comment by mrshah-at-work** [comment]
The addition of wait for the utils container in https://github.com/IBM-Blockchain/ibm-container-service/pull/19/files#diff-9e265e39cd55927799fdf2fa455386d4R80 should fix the issue.. @dwakeman can you please verify?

**Comment by dwakeman** [comment]
I'm not sure how to answer.  I created a new cluster, deleted my local repo, cloned this repo (today), then ran `./create_all.sh --with-couchdb` and I think it worked.  I'm not sure if this change had anything to do with it though.  I noticed that I updated the delay back to 120 seconds (I was also using these scripts against a completely different cluster), so I ran the following commands to clean up my cluster:
```
kubectl delete deployments --all
kubectl delete pods --all
kubectl delete svc --all
kubectl delete pv --all
kubectl delete pvc --all
```

Then I ran the script again, and the `createchannel` pod failed again.  This is the content of the logs of the `createchannel` container in the pod:
```kubectl logs createchannel createchannel
2017-10-19 01:34:32.589 UTC [msp] GetLocalMSP -> DEBU 001 Returning existing local MSP
2017-10-19 01:34:32.589 UTC [msp] GetDefaultSigningIdentity -> DEBU 002 Obtaining default signing identity
2017-10-19 01:34:32.643 UTC [channelCmd] InitCmdFactory -> INFO 003 Endorser and orderer connections initialized
2017-10-19 01:34:32.645 UTC [msp] GetLocalMSP -> DEBU 004 Returning existing local MSP
2017-10-19 01:34:32.645 UTC [msp] GetDefaultSigningIdentity -> DEBU 005 Obtaining default signing identity
2017-10-19 01:34:32.645 UTC [msp] GetLocalMSP -> DEBU 006 Returning existing local MSP
2017-10-19 01:34:32.646 UTC [msp] GetDefaultSigningIdentity -> DEBU 007 Obtaining default signing identity
2017-10-19 01:34:32.646 UTC [msp/identity] Sign -> DEBU 008 Sign: plaintext: 0A8C060A074F7267314D53501280062D...53616D706C65436F6E736F727469756D 
2017-10-19 01:34:32.646 UTC [msp/identity] Sign -> DEBU 009 Sign: digest: D23A33DDCB8B5EE9755B34D7C033FA00663697D20167C8E0429FA3EA646242A2 
2017-10-19 01:34:32.647 UTC [msp] GetLocalMSP -> DEBU 00a Returning existing local MSP
2017-10-19 01:34:32.647 UTC [msp] GetDefaultSigningIdentity -> DEBU 00b Obtaining default signing identity
2017-10-19 01:34:32.647 UTC [msp] GetLocalMSP -> DEBU 00c Returning existing local MSP
2017-10-19 01:34:32.647 UTC [msp] GetDefaultSigningIdentity -> DEBU 00d Obtaining default signing identity
2017-10-19 01:34:32.648 UTC [msp/identity] Sign -> DEBU 00e Sign: plaintext: 0AC2060A1408021A0608A882A0CF0522...0D8406F74F1B42274B0975686B3C5D97 
2017-10-19 01:34:32.648 UTC [msp/identity] Sign -> DEBU 00f Sign: digest: 09572C69E616E4CE4E846AB58E6736434584141B27C7506C89DFB7FECC2B6FCF 
Error: Got unexpected status: BAD_REQUEST
Usage:
  peer channel create [flags]

Flags:
  -c, --channelID string   In case of a newChain command, the channel ID to create.
  -f, --file string        Configuration transaction file generated by a tool such as configtxgen for submitting to orderer
  -t, --timeout int        Channel creation timeout (default 5)

Global Flags:
      --cafile string              Path to file containing PEM-encoded trusted certificate(s) for the ordering endpoint
      --logging-level string       Default logging level and overrides, see core.yaml for full syntax
  -o, --orderer string             Ordering service endpoint
      --test.coverprofile string   Done (default "coverage.cov")
      --tls                        Use TLS when communicating with the orderer endpoint
  -v, --version                    Display current version of fabric peer server
```
Something must be wrong with the way I am cleaning things up, I guess.  I deleted my cluster and am in the process of creating a new one to verify that it works with a newly-created cluster.

**Comment by dwakeman** [comment]
Update on my progress.  I ran `create_all.sh` against a squeaky clean brand new cluster and it seems to have worked better.  This time my composer-playground pod was in a CrashBackoffLoop, so I deleted the pod and copy/pasted the command the script ran to create it and now it seems to work.  I can access composer-playground, so it seems to be okay.  Not sure what the problem was, but it wasn't related to this issue.
